src/entries/web-compiler.js

```diff
@@ -16,6 +17,8 @@ const cache2: { [key: string]: CompiledFunctions } = Object.create(null)
 const baseOptions: CompilerOptions = {
   expectHTML: true,
   preserveWhitespace: true,
+  modules,
+  staticKeys: genStaticKeys(modules),
```

Comment:
actually this option can be a function called `isStaticKey` directly... no need to cache it!